backend/tests/results/test_apps.json

```diff
@@ -18,6 +18,6 @@
     "name": "WPS Office",
     "summary": "WPS Office Suite",
     "type": "desktop-application",
-    "icon": "https://www.wps.com/assets/mediahub/WPS-Office-Mobile.png"
+    "icon": null
```

Comment:
This is due to remote icons being a no go now?

Reply:
Yes, apps with remote icons built with appstream-glib will no longer have this defined. The ones that used libappstream will have this properly mirrored to our infra